xinference/model/llm/utils.py

```diff
@@ -174,6 +174,9 @@ def get_prompt(
                 else:
                     ret += role + ": Let's think step by step."
             return ret
+        elif prompt_style.style_name == "INSTRUCTION":
```

Comment:
Multi-round conversation is not supported？

Reply:
Yes. Multi-round is not good for translate.